app/jobs/models-utils.js

```diff
@@ -102,7 +102,7 @@ var job = {
                     issue.assignPerson = element.fields.assignee.name + "/" + element.fields.assignee.emailAddress;
                 }
                 else {
-                    issue.assignPerson = "";
+                    issue.assignPerson = "Unassigned";
```

Comment:
keep it as constant...place all const in yml file...

Reply:
done